### PR TITLE
docs(); production errors

### PR DESCRIPTION
**Pull request by dmelotik**
Describe protocol for production subgraph errors

### CODE REVIEW

File: docs/ProductionErrors.md [hunk 1]
```diff
@@ -1 +1,60 @@
...
+2. Mute the channel.
+3. Jump into the `Production Ready Subgraph Indexing Failures` thread.
+4. Unmute this thread.
+5. Congrats! Now you will get notified of production errors here.
+
+## Fixing Errors
+
+> As these subgraphs are in production we want to be efficient and solve the problems swiftly to reduce downtime.
+
+You will need to use your subgraph debugging skills, but here are some tips to help!
```
Comment (tnkrxyz) [inline docs/ProductionErrors.md:40]: Add a link to https://github.com/messari/subgraphs/blob/master/docs/Mapping.md#debugging ?

**Comment by tnkrxyz** [comment]
LGTM; I can approve. Not sure if others have inputs.

**Comment by this-username-is-taken** [comment]
@tnkrxyz just wondering, do you have permission to formally approve on Github?
image

**Review by tnkrxyz — APPROVED** [review]

**Comment by dmelotik** [comment]
Looks like yes @this-username-is-taken

**Pull request merged**